test/test_helper.rb

```diff
@@ -62,6 +62,10 @@ def set_basic_auth(user, password)
     @request.env['HTTP_ACCEPT'] = 'application/json'
   end
 
+  def fetch_proxy_features
+    ProxyAPI::Features.any_instance.stubs(:features => Feature.name_map.keys)
```

Comment:
Wasn't this going to be removed now? (https://github.com/theforeman/foreman/pull/3895#discussion_r81988688)